### ISSUE TITLE
[a11y][Violation][1.3.1 Info and Relationships] Issues reported on Notebook UI using IBM Equal Access Accessibility Checker tool

### ISSUE THREAD

**Issue by nkn2022** [jupyter/notebook#7491]
## Description

I am running an accessibility test suite with Notebook UI using the **IBM Equal Access Accessibility Checker** tool on Chrome browser. The results look good overall, only a few things are missing to reach a first formal level of compliance. So kindly help look into these. 

Issues reported were violating **WCAG 2.2** requirements. 
Ref: https://www.ibm.com/able/requirements/checker-rule-sets

Full report (includes information on element locations where issue were found): 
[Accessibility_Report-7a73e10e-f8a4-4a06-a78d-5174a1309c5b.xlsx](https://github.com/user-attachments/files/17400350/Accessibility_Report-7a73e10e-f8a4-4a06-a78d-5174a1309c5b.xlsx)

# Category: `1.3.1 Info and Relationships`
Ref: https://www.ibm.com/able/requirements/requirements/?version=v7_3#1_3_1

****Violation of Rule:** `table_headers_related`**

> List of issues with elements that violate this rule with a screenshots
> > **Issue 1** 
> > Complex table does not have headers for each cell properly defined with 'header' or 'scope'
> > **Element location:** 
> > `<th>`
> > **Screenshot:**
> > ![Image](https://github.com/user-attachments/assets/acdfd246-cbaa-4958-ac05-440dd2216ae7)

## Reproduce

- Install Chrome extension for IBM accessibility checker tool. Ref: https://www.ibm.com/able/toolkit/verify/automated
- Open Notebook UI on Chrome browser. Add some cells with code.
- Run the tool and we can see the above `Violation`s reported.

Here is the screenshot of the UI scanned for the attached report:
![Image](https://github.com/user-attachments/assets/895eebec-51ad-43cd-93bb-99c4a8193b68)

## Expected behavior

No violations are reported on the scanning to reach a first formal level of compliance.

**Comment by nkn2022** [comment]
I do not see this issue is reported anymore. Don't see any fixes in this area so guessing the requirements may have changed and the tool do not raised this as a critical issue. 
Closing this issue.
